src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -47,6 +47,303 @@
         </code>
       </example>
     </member>
-  
+
+    <member name="CountFeatureSelection">
+      <summary>
+        Selects the slots for which the count of non-default values is greater than or equal to a threshold.
+      </summary>
+      <remarks>
+        <para>
+          This transform uses a set of aggregators to count the number of non-default values for each slot and
+          instantiates a <see cref="Microsoft.ML.Runtime.Data.DropSlotsTransform"/> to actually drop the slots.
+          This transform is useful when applied together with a CategoricalHashOneHotVectorizer. 
```

Comment:
>CategoricalHashOneHotVectorizer [](start = 64, length = 31)

Should this have a `<see` tag somehow?

Reply:
agree, but the compiler was complaining of not being able to find a reference to that class, although the cref of the see tag had absolute reference to the CategoricalHashOneHotVectorizer, starting from the namespace. 
I'll try to get to the bottom of the hiccup was. 

---
In reply to: [202461085](https://github.com/dotnet/machinelearning/pull/529#discussion_r202461085) [](ancestors = 202461085)

Reply:
@sfilipi There are a few ways to address this if you want the link to resolve but don't want to mess up code references. My favorite tends to be declaring an assembly reference but specifying `<Aliases>` to ensure the reference can't be readily used in code. This form allows for a compiler-validated `cref` that doesn't result in an assembly reference in the compiled assembly. I'll try to find an example.

A secondary way to resolve this is using a fully-expanded cref that the compiler simply passes through without validation, but that documentation tools know how to parse:

```xml
<!-- This is inferior because the compiler will not validate it, but works as a last resort -->
<see cref="T:Fully.Qualified.Path.To.TypeName" />
```

Reply:
That's what was not working, actually.
 I didn't specify the T: in the cref. I'll try that. 
Thanks for the suggestion, though. 

---
In reply to: [202780648](https://github.com/dotnet/machinelearning/pull/529#discussion_r202780648) [](ancestors = 202780648)